bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gems/Canopus.java

```diff
@@ -227,154 +152,99 @@ public GuideStarValidation validate(SPTarget guideStar, ObsContext ctx) {
     private static final Point2D.Double PROBE1_CENTER = new Point2D.Double(0.11 * 60.0, 0.3 * 60.0);
     private static final Point2D.Double PROBE2_CENTER = new Point2D.Double(0.0, -0.33 * 60.0);
     private static final Tuple2<Double, Double> PROBE1_DIM = new Pair<>(3.7 * 60.0, 3.0 * 60.0);
-
-        //     REL-1042: Update the Canopus probe limits in the OT
-//    private static final Tuple2<Double, Double> PROBE2_DIM = new Pair<Double, Double>(4.2 * 60.0, 3.0 * 60.0);
     private static final Tuple2<Double, Double> PROBE2_DIM = new Pair<>(4.2 * 60.0, 2.5 * 60.0);
-
     private static final Ellipse2D AO_PORT = new Ellipse2D.Double(-RADIUS_ARCSEC, -RADIUS_ARCSEC, RADIUS_ARCSEC*2, RADIUS_ARCSEC*2);
 
-    // Width of CWFS probe arm in arcsec
-//    private static final double PROBE_ARM_WIDTH = 12.95;
-    private static final double PROBE_ARM_WIDTH = 10.0; // see REL-160
-
-    // Probe arm end, as distance in arcsec from the CWFS guide star
-    private static final double PROBE_ARM_END = 8.9;
-
-
     /**
      * Returns a Shape that defines the AO port in arcsecs, centered at
      * <code>(0,0)</code>.
      */
     private static final Shape AO_PORT_SHAPE = new Ellipse2D.Double(AO_PORT.getX(), AO_PORT.getY(), AO_PORT.getWidth(), AO_PORT.getHeight());
 
     // Gets the primary CWFS 3 guide star, if any.
-    private Option<SPTarget> getPrimaryCwfs3(ObsContext ctx) {
-        TargetEnvironment env = ctx.getTargets();
-        if (env == null) return None.instance();
-
-        Option<GuideProbeTargets> gtOpt = env.getPrimaryGuideProbeTargets(Canopus.Wfs.cwfs3);
-        if (gtOpt.isEmpty()) return None.instance();
-        return gtOpt.getValue().getPrimary();
+    private Option<SPTarget> getPrimaryCwfs3(final ObsContext ctx) {
+        return ImOption.apply(ctx.getTargets())
+                .flatMap(env -> env.getPrimaryGuideProbeTargets(Wfs.cwfs3))
+                .flatMap(GuideProbeTargets::getPrimary);
     }
 
     // Gets the coordinates of the primary CWFS guide star relative to the
     // base position, translated to screen coordinates.
-    private Option<Point2D> getPrimaryCwfs3Offset(ObsContext ctx) {
-        Option<SPTarget> spTargetOpt = getPrimaryCwfs3(ctx);
-        if (spTargetOpt.isEmpty()) return None.instance();
-
-        Asterism asterism = ctx.getTargets().getAsterism();
-        SPTarget target   = spTargetOpt.getValue();
-
-        final Option<Long> when = ctx.getSchedulingBlockStart();
-
-        return
-            asterism.getSkycalcCoordinates(when).flatMap(bc ->
-            target.getSkycalcCoordinates(when).map(tc -> {
-                CoordinateDiff diff = new CoordinateDiff(bc, tc);
-                Offset o = diff.getOffset();
-                double p = -o.p().toArcsecs().getMagnitude();
-                double q = -o.q().toArcsecs().getMagnitude();
-                return new Point2D.Double(p, q);
-            }));
-    }
+    private Option<Point2D> getPrimaryCwfs3Offset(final ObsContext ctx) {
```

Comment:
The reason for doing this was to define the probe ranges for CWFS1 and CWFS2 -- they moved along with CWFS3.  Maybe we don't need this at all anymore?